nightly/fuzzing.txt

```diff
@@ -1,2 +1,5 @@
-pytest --skip-build --timeout=2h runtime/fuzz_runtime.py
-pytest --skip-build --timeout=2h runtime/fuzz_wasm_vm.py
+pytest --skip-build --timeout=2h fuzz.py test-utils/runtime-tester/fuzz runtime-fuzzer
```

Comment:
2h timeout ? is that intended ? How long are these tests running ?

Reply:
Yes. The test run as long as we give them. It’s fuzzing so it will continue trying random inputs until failure is found or we kill it.